dll/win32/comctl32/taskdialog.c

```diff
@@ -320,7 +320,7 @@ static unsigned int taskdialog_add_buttons(struct taskdialog_template_desc *desc
     /* Try to balance lines so they are about the same size */
     for (i = 1; i < line_count - 1; i++)
     {
-        int diff_now = abs(line_widths[i] - line_widths[i - 1]);
+        int diff_now = line_widths[i] - line_widths[i - 1];
```

Comment:
Code change in Wine code: Surround the modification in `#ifdef __REACTOS__  ... #endif`

Reply:
Yes, will do that later.